### ISSUE TITLE
Logging improvements

### ISSUE THREAD

**Issue by richteer** [Halibot/halibot#46]
Right now, the log initialized in `main.py` always spews to `stdout`. There should be some way either by configuration or via the command line to specify a file for the log to write to instead (particularly useful if running interactively (`python3 -i`)).

Also, the log is stuck at a hardcoded level. It should also be configurable what level of detail should be outputted (DEBUG vs INFO, etc.)

Examples of the usage:

```
python3 main.py --logfile foo.log --loglevel DEBUG
```

...or in `config.json`:

``` json
{
    "logfile": "foo.log",
    "loglevel": "DEBUG",
...
}
```

**Comment by richteer** [comment]
This could follow a solution to #57, where `config.json` should be loaded in `main.py` before the logger is initialized.

**Comment by richteer** [comment]
Implemented in PR #63
